packages/host/webpack.config.js

```diff
@@ -1,7 +1,7 @@
 const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
 const webpack = require('webpack');
-const HotModuleReplacementPlugin = webpack.HotModuleReplacementPlugin;
```

Comment:
as per the warning when run: HMR is automatically added when `hot: true` is used